<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="playButton">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>10</y>
           <width>115</width>
           <height>32</height>
          </rect>
         </property>
         <property name="text">
          <string>Play</string>
         </property>
        </widget>
        <widget class="QPushButton" name="stopButton">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>10</y>
           <width>115</width>
           <height>32</height>
          </rect>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
        <widget class="QSlider" name="speedSlider">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>10</y>
           <width>160</width>
           <height>22</height>
          </rect>
         </property>
         <property name="sliderPosition">
          <number>49</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionTextures">
   <property name="text">
    <string>Textures</string>
   </property>
  </action>
  <action name="action_clairage">
   <property name="text">
    <string>Ã‰clairage</string>
   </property>
  </action>
  <action name="actionMag_Filter">
   <property name="text">
    <string>Mag Filter</string>
   </property>
  </action>
  <action name="actionNearest">
   <property name="text">
    <string>Nearest</string>
   </property>
  </action>
  <action name="actionLinear_mipmap_linear">
   <property name="text">
    <string>Linear mipmap linear</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
